src/components/TokenCard/TokenCard.js

```diff
@@ -37,9 +37,9 @@ export default function TokenCard({ showRedirectModal }) {
   const BuyLink = ({ className, to, children, network }) => {
     if (isHome && showRedirectModal) {
       return (
-        <div className={cx("a", className)} onClick={() => showRedirectModal(to)}>
+        <HeaderLink to={to} className={cx(className)}>
```

Comment:
i guess HeaderLink is intended to be used inside Header? 

HeaderLink was moved to separate file so it can be imported, no need to pass it as a prop
i think we can rename it (and move away from Header component) and reuse everywhere in other components

Reply:
Yes for sure. I will update based on the new refactoring. HeaderLink contains the validation for opening the Launch App modal. So, I was using it here.

Reply:
Done!

Reply:
HeaderLink now required props showRedirectModal and redirectPopupTimestamp (Previously, these variables were taken from the scope of App.js, but it was implicit dependency)